crates/krun-guest/src/bin/krun-guest.rs

```diff
@@ -37,14 +37,16 @@ fn main() -> Result<()> {
 
     setup_pulse_proxy(run_path)?;
 
+    let command = match options.command {
+        Some(c) => c,
+        None => "/usr/local/bin/krun-server".to_string(),
+    };
```

Comment:
```suggestion
    let command = options.command.unwrap_or("/usr/local/bin/krun-server".to_owned());
```

Reply:
In which sense? As an option to specific the name of the `krun-server`?

Reply:
This is using `krun-server` as the default `command`, right? What I meant is that we could set this fallback value in `cli_options` instead of here.

Reply:
This was a leftover from a previous approach. I've dropped it.